.github/CODEOWNERS

```
@@ -361,6 +361,7 @@
 /comp/metadata/haagent @DataDog/ndm-core
 /comp/metadata/hostgpu @DataDog/ebpf-platform
 /comp/metadata/packagesigning @DataDog/agent-delivery
+/comp/networkpath/traceroute @DataDog/cloud-network-monitoring
```

Can you add the Network Path team here as well please?

```suggestion
/comp/networkpath/traceroute @DataDog/cloud-network-monitoring @DataDog/network-path 
```